zerver/webhooks/teamcity/doc.md

```diff
@@ -1,37 +1,38 @@
-See your Teamcity build status in Zulip!
+Get Zulip notifications for your Teamcity builds!
 
-{!create-stream.md!}
+1. {!create-stream.md!}
 
-{!create-bot-construct-url.md!}
+1. {!create-bot-construct-url-indented.md!}
 
-Next, install the
-[tcWebHooks plugin](https://github.com/tcplugins/tcWebHooks/releases)
-onto your Teamcity server. Follow the plugin instructions in your
-Teamcity documentation, or refer to [the online Teamcity documentation][1].
+1. Install the [tcWebHooks plugin](https://github.com/tcplugins/tcWebHooks/releases)
+   onto your Teamcity server. Follow the plugin instructions in your
+   Teamcity documentation, or refer to [the online Teamcity documentation][1].
 
-[1]: https://confluence.jetbrains.com/display/TCD9/Installing+Additional+Plugins
-
-Next, in your Teamcity project overview page, click the **Webhooks**
-tab, and add a new project webhook. Enter the URL you created
-earlier.
+1. Go to your TeamCity **Overview** page. Select the **Project** or **Build**
+   you'd like to receive notifications about, and click on the **WebHooks** tab.
+   If you'd like to configure webhooks for a **Project**, click on
+   **Add project WebHooks**. If you'd like to configure webhooks for a specific
+   **Build**, click on **Add build WebHooks**. Click on
+   **Click to create new WebHook for this project/build**.
 
-Uncheck all **Trigger on Events** options, and check
-**Trigger when build is Successful** and **Trigger when build Fails**.
-Optionally, check **Only trigger when build changes from Failure to Success**
-and **Only trigger when build changes from Success to Failure**.
+1. Set **URL** to the URL constructed above. Set **Payload Format** to
+   **Legacy Webhook (JSON)**. Uncheck all **Trigger on Events** options,
+   and check **Trigger when build is Successful** and **Trigger when build Fails**.
+   Optionally, check **Only trigger when build changes from Failure to Success**
```

Comment:
We could split this ("Optionally, ..") into it's own instruction. Not what we typically do, but this block in particular is pretty dense.